murano/api/v1/templates.py

```diff
@@ -138,11 +142,60 @@ def delete(self, request, template_id):
         temps.TemplateServices.remove(template_id)
         return
 
+    def has_services(self, template):
+        if isinstance(template.description['Objects'], types.DictionaryType):
```

Comment:
Currently, the template description does not have Object. You can have a look at the spec